src/app/admin/common/passwordConfig.ts

```diff
@@ -0,0 +1,5 @@
+export const passwordRuleDisplay = "Password length must be six characters or longer.";
+
+export const passwordRule = (password: string): boolean => {
+    return password.length > 6;
```

Comment:
I realise that this is mostly a placeholder right now but is it worth standardising an interface for password requirements so adding more doesn't require refactoring?